src/main/java/uk/gov/hmcts/darts/arm/util/ArmResponseFilesHelper.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.darts.arm.util;
+
+import lombok.experimental.UtilityClass;
+import uk.gov.hmcts.darts.common.entity.ExternalObjectDirectoryEntity;
+
+import static java.util.Objects.nonNull;
+import static uk.gov.hmcts.darts.arm.util.ArchiveConstants.ArchiveRecordOperationValues.ARM_FILENAME_SEPARATOR;
+import static uk.gov.hmcts.darts.arm.util.ArchiveConstants.ArchiveResponseFileAttributes.ARM_RESPONSE_FILE_EXTENSION;
+
+@UtilityClass
+@SuppressWarnings({"HideUtilityClassConstructor"})
+public class ArmResponseFilesHelper {
+
+    public static String generateSuffix(String filenameKey) {
+        return ARM_FILENAME_SEPARATOR + filenameKey + ARM_RESPONSE_FILE_EXTENSION;
```

Comment:
is it worth using stringbuilder for this one as in getPrefix()?

Reply:
Intellij complains about using string builder for this. Also its just adding 3 strings which I would consider the max for strings being concatenated